Spreadsheet/style.css

```diff
@@ -74,29 +73,26 @@ header {
     overflow: scroll;
 }
 
-.box {
+.cell {
```

Comment:
`cell` just felt more appropriate here as it's the name used in most table/sheet editors. `box` sounds more like a block in the page, like a card or a container around something.